examples/viewer.js

```diff
@@ -991,8 +1014,10 @@ function calcStats(stats, clientId) {
                 htmlString =
                     '<table><tr><strong>DQP TEST (2min) - <FONT COLOR=RED>RESULTS READY IN: ' + (DQPtestLength - statRunTime) + ' sec</FONT></strong></tr>' +
                     '<tr><td>Client ID: </td><td>' + clientId + '</td></tr>' +
-                    '<tr><td>Time to P2P connection: </td><td>' + connectionTime + ' sec</td></tr>' +
```

Comment:
We can still calculate the time to P2P connection using `metrics.viewer.peerConnection.endTime - metrics.viewer.peerConnection.startTime`

Reply:
This time to P2P was incorrect (was calculating time to adding track). So removed it altogether and split it into meaningful profile data in this PR.

Reply:
I mean we can correctly calculate it now

Reply:
We dont need to calculate time to add track. We are calculating time taken for different steps. Is there a specific metric you are talking about?